pipeline/metadata/6.0.yaml

```diff
@@ -7,6 +7,22 @@
 # parameter required to create a similar custom file is suite name, suite yaml file, global configuration file,
 # platform, rhbuild, inventory and metadata information like frequency of execution, tier, cloud type, functional group and stage.
 #=====================================================================================
+- name: "Build smoke test for 6.0"
```

Comment:
nit: Could we rename the test suite name `Smoke Test of RH Ceph build`

Reply:
Done.